integration/jnifuse/fs/src/main/java/alluxio/jnifuse/AbstractFuseFileSystem.java

```diff
@@ -37,7 +36,6 @@
 public abstract class AbstractFuseFileSystem implements FuseFileSystem {
 
   static {
-    LibFuse.loadLibrary(VersionPreference.NO);
```

Comment:
We loaded the library in `launchFuse()` in `AlluixoFuse.java`, so no need to do it here. The alluxio fuse filesystem should not care about the version of libfuse.